Gopkg.toml

```diff
@@ -64,6 +64,13 @@ required = [
   name="sigs.k8s.io/controller-tools"
   version="v0.1.1"
 
+# Explicitly require the switch from glog to klog otherwise we get
+# glog and klog registering the same flag which will then panic.
+[[override]]
+  name = "k8s.io/gogen"
+  source = "https://github.com/kubernetes/gogen.git"
```

Comment:
https://github.com/kubernetes/gogen does not exists

Reply:
> https://github.com/kubernetes/gogen does not exists

PEBCAK. :-/